oops/hosts/voyager/iss.py

```
@@ -48,19 +47,16 @@ def from_file(filespec, astrometry=False, action='error', parameters={}):
             labelspec = filespec.with_suffix('.lbl')
 
         try:
-            local_labelspec = labelspec.retrieve()
+            label_dict = pdsparser.Pds3Label(labelspe, method=method).as_dict()
```

Ruff (0.15.12)

[error] Undefined name `labelspe`

(F821)